src/Microsoft.OData.Edm/EdmUtil.cs

```diff
@@ -260,8 +261,10 @@ internal static string ParameterizedName(IEdmOperation operation)
             return sb.ToString();
         }
 
-        internal static bool TryGetNamespaceNameFromQualifiedName(string qualifiedName, out string namespaceName, out string name)
+        internal static bool TryGetNamespaceNameFromQualifiedName(string qualifiedName, out string namespaceName, out string name, out string fullName)

```

Comment:
>TryGetNamespaceNameFromQualifiedName [](start = 29, length = 36)

Rather than requiring callers that don't care about full name (i.e., FindVocabularyAnnotations and CsdlSemanticsModel.AddSchema (which iterates over every element in the schema) to pass an output variable and compute a value that they never use, should we instead just create a separate overload to return the fullname for those that will use it?

Reply:
I feel this method is the right place to get the fullname as well since it's already returning the namespace and name and getting the full name feels like the logical thing to do here.